envs/kickstarter/messages/it/app.php

```diff
@@ -0,0 +1,7 @@
+<?php
```

Comment:
This file has been commite already inside another PR (https://github.com/luyadev/luya/commit/47930f67fac3a450e2c334597a609b9fa983132b)